src/lib/ui/components/File/File.svelte

```diff
@@ -38,22 +38,28 @@
   $: datasets = generateDatas(datas);
 
   function onSelectFile(event: Event) {
-    const { files: filesToUpload } = event.target as HTMLInputElement;
-    if (filesToUpload) {
-      const fileToUpload =
+    const { files: filesInput } = event.target as HTMLInputElement;
+    if (filesInput) {
+      const filesToUpload =
         multiple && files
-          ? ([...files, ...filesToUpload] as File[])
-          : filesToUpload[0];
+          ? ([...files, ...filesInput] as File[])
+          : filesInput[0];
 
-      if (fileToUpload instanceof File) {
-        const sizeKB = fileToUpload.size / 1024;
+      if (filesToUpload instanceof File || Array.isArray(filesToUpload)) {
+        const sizeKB =
+          (filesToUpload instanceof File
```

Comment:
🚫 **[eslint]** <[operator-linebreak](https://eslint.org/docs/latest/rules/operator-linebreak)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>There should be no line break before or after '='.
```suggestion
        const sizeKB =          (filesToUpload instanceof File
```